src/main/kotlin/racingcar/view/ResultView.kt

```diff
@@ -1,32 +1,26 @@
 package racingcar.view
 
-import racingcar.domain.Car
-import racingcar.domain.Cars
+import racingcar.domain.car.Car
+import racingcar.domain.car.Cars
 
-class ResultView(private val cars: Cars) {
+class ResultView {
 
-    fun print() {
+    fun printTitle() {
         println("실행 결과")
-        printEachTurn()
-        printWinner()
     }
 
-    private fun attemptCount() = cars[0].history.size
-
-    private fun printEachTurn() {
-        for (i in 1..attemptCount()) {
-            cars.forEach { printCar(it, i) }
-            println()
-        }
+    fun printEachTurn(cars: Cars) {
+        cars.forEach { printCar(it) }
+        println()
     }
 
-    private fun printCar(car: Car, attempt: Int) {
-        print("${car.name} : ")
-        println("-".repeat(car.currentPosition(attempt)))
+    fun printWinner(cars: Cars) {
+        val winners = cars.winners()
+        println("${winners.joinToString(", ") { it.name }}가 최종 우승했습니다.")
     }
 
-    private fun printWinner() {
-        val winners = cars.winners(attemptCount())
-        println("${winners.joinToString(", ") { it.name }}가 최종 우승했습니다.")
+    private fun printCar(car: Car) {
+        print("${car.name} : ")
+        println("-".repeat(car.movedDistance()))
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        println("${car.name} : ${"-".repeat(car.movedDistance())}")
```

Reply:
https://github.com/next-step/kotlin-racingcar/pull/238/commits/e96f7f70b445db67998bec2fe74cc2c7b9107446 반영했습니다.